src/main/java/hu/oe/nik/szfmv/automatedcar/model/World.java

```diff
@@ -1,44 +1,101 @@
 package hu.oe.nik.szfmv.automatedcar.model;
 
+import java.awt.*;
```

Comment:
*-al ne importáljunk semmit, checkstyle

Reply:
Az IDEA importálta automatikusan, mint láthatod nincs is használva.